service/application.go

```diff
@@ -139,7 +141,11 @@ func New(params Parameters) (*Application, error) {
 	parserProvider := params.ParserProvider
 	if parserProvider == nil {
 		// use default provider.
-		parserProvider = parserprovider.Default()
+		logger, err := zap.NewDevelopment() // TODO find a way to share with app.
```

Comment:
@bogdandrutu that is happening this way because the logger is not initialized at this point.

Reply:
also, the logger should be passed via the interface itself.